cocos/core/pipeline/forward/forward-pipeline.ts

```diff
@@ -345,6 +345,19 @@ export class ForwardPipeline extends RenderPipeline {
         }
     }
 
+    /**
+     * @en Map the value of [-1,1] in screenSpaceSignY and clipSpaceSignY to [0,1],and then move screenSpaceSignY one bit to the left and combine with clipSpaceSignY
+     * @zh 把screenSpaceSignY与clipSpaceSignY中[-1,1]的值映射为[0,1],通过screenSpaceSignY左移一位与clipSpaceSignY合并
```

Comment:
keep the comments simple and stupid:

|combinedSignY|clipSpaceSignY|screenSpaceSignY|
|    :--:     |    :--:      |      :--:      |
|      0      |      -1      |      -1        |
|      1      |       1      |      -1        |
|      2      |      -1      |       1        |
|      3      |       1      |       1        |